Changelog.md

```diff
@@ -53,6 +60,22 @@
     * `encodeUtf8Lax` to `encodeUtf8`
     * `encodeLatin1Lax` to `encodeLatin1`
     * `decodeUtf8Lenient` to `decodeUtf8`
+* Drop support for GHC 7.10.3.
+
+### Internal APIs
+
+* `Streamly.Internal.Prelude` renamed to `Streamly.Internal.Data.Stream.IsStream`
+* Parser functionality is added via `Streamly.Internal.Data.Parser` module
+* `Streamly.Internal.Data.Binary.Decode` module added for decoding Haskell
+   values from binary data.
+* FileSystem event notification (fsnotify/inotify) functionality added via
+  `Streamly.Internal.FileSystem.Event.*` modules.
+* The `Fold` type has changed to accomodate terminating folds.
+* Added `use-c-malloc` build flag to use the c library `malloc` for array
+  allocations.
+* A bug was fixed in the conversion of MicroSecond64 and MilliSecond64
+  (commit e5119626)
+* Bug fix: classifySessionsOf now flushes sessions at the end and terminates.
```

Comment:
Making this statement about `classifySessionsBy` makes more sense?

Reply:
Yes.